packages/indexer-agent/src/agent.ts

```diff
@@ -609,40 +619,41 @@ class Agent {
       epoch,
     })
 
-    const allocationAmount = rule?.allocationAmount
+    const desiredAllocationAmount = rule?.allocationAmount
       ? BigNumber.from(rule.allocationAmount)
       : this.indexer.defaultAllocationAmount
     const desiredNumberOfAllocations = Math.max(
       1,
       rule?.parallelAllocations || 1,
     )
+    const activeAllocationAmount = activeAllocations.reduce(
+      (sum, allocation) => sum.add(allocation.allocatedTokens),
+      BigNumber.from('0'),
+    )
 
-    logger.info(`Reconcile deployment allocations`, {
-      allocationAmount: formatGRT(allocationAmount),
+    logger.trace(
+      `Reconcile deployment allocations for deployment '${deployment.ipfsHash}'`,
+      {
+        desiredAllocationAmount: formatGRT(desiredAllocationAmount),
 
-      totalActiveAllocationAmount: formatGRT(
-        activeAllocations.reduce(
-          (sum, allocation) => sum.add(allocation.allocatedTokens),
-          BigNumber.from('0'),
-        ),
-      ),
+        totalActiveAllocationAmount: formatGRT(activeAllocationAmount),
 
-      desiredNumberOfAllocations,
-      activeNumberOfAllocations: activeAllocations.length,
+        desiredNumberOfAllocations,
+        activeNumberOfAllocations: activeAllocations.length,
 
-      activeAllocations: activeAllocations.map(allocation => ({
-        id: allocation.id,
-        createdAtEpoch: allocation.createdAtEpoch,
-        amount: formatGRT(allocation.allocatedTokens),
-      })),
-    })
+        activeAllocations: activeAllocations.map(allocation => ({
+          id: allocation.id,
+          createdAtEpoch: allocation.createdAtEpoch,
+          amount: formatGRT(allocation.allocatedTokens),
+        })),
+      },
+    )
 
     // Return early if the deployment is not (or no longer) worth indexing
     if (!worthIndexing) {
       logger.info(
-        `Deployment is not (or no longer) worth indexing, close all active allocations that are at least one epoch old`,
+        `Deployment is not (or no longer) worth indexing, please close allocation if it is from a previous epoch`,
```

Comment:
This log message isn't asking the user to do anything, rather letting the user know it is about to close allocations.  

Maybe something like `Deployment is not (or no longer) worth indexing, closing all active allocations that are at least one epoch old`?

Reply:
I was confused by `at least one epoch old`. An allocation that opened last 20 minutes of epoch2 can technically be closed within 30 minutes of it opened, right?

Reply:
O interesting point. One epoch old does imply it has been around for at least 1 epoch. 

How about, `Deployment is not (or no longer) worth indexing, close allocation if eligible`?
or  ``Deployment is not (or no longer) worth indexing, close allocation if opened in a previous epoch`?